Pradeo/pradeo-mtd/ingest/parser.yml

```
@@ -0,0 +1,209 @@
+          pradeo.device.coupled: "{{json_event.message.content.deviceNetworkStatusRecord.device.enrollmentStatus.coupled}}"
+          pradeo.device.mdmId: "{{json_event.message.content.deviceNetworkStatusRecord.device.emmDeviceInfo.externalId}}"
+          pradeo.device.emm: "{{json_event.message.content.deviceNetworkStatusRecord.device.emmDeviceInfo.emm}}"
+          pradeo.detection.status: "{{json_event.message.content.deviceNetworkStatusRecord.deviceNetworkStatus.name}}"
+          pradeo.detection.value: "{{json_event.message.content.deviceNetworkStatusRecord.value}}"
+      - filter: '{{json_event.message.type == "DeviceStatusHistoryUpdated"}}'
+        set:
+          event.category: "process"
+          event.type: "info"
+          pradeo.device.id: "{{json_event.message.content.deviceId}}"
+          pradeo.detection.status: "{{json_event.message.content.event.kind}}"
+          pradeo.detection.value: "{{json_event.message.content.event.value}}"
+  correlation:
+    actions:
+      - filter: '{{json_event.message.type == "DeviceCorrelationUpdated"}}'
+        set:
+          event.category: "process"
+          event.type: "change"
+          pradeo.device.id: "{{json_event.message.content.deviceCorrelation.device.id}}"
+          pradeo.device.serialNumber: "{{json_event.message.content.deviceCorrelation.device.serialNumber}}"
+          pradeo.device.name: "{{json_event.message.content.deviceCorrelation.device.name}}"
+          pradeo.device.email: "{{json_event.message.content.deviceCorrelation.device.email}}"
+          pradeo.device.imei: "{{json_event.message.content.deviceCorrelation.device.imei}}"
+          pradeo.device.byod: "{{json_event.message.content.deviceCorrelation.device.byod}}"
+          pradeo.device.declaredOperatingSystem: "{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystem}}"
+          pradeo.device.declaredOperatingSystemVersion: "{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystemVersion}}"
+          pradeo.device.declaredOperatingSystemSecurityPatchDate: "{{json_event.message.content.deviceCorrelation.device.declaredOperatingSystemSecurityPatchDate}}"
+          pradeo.device.declaredModel: "{{json_event.message.content.deviceCorrelation.device.declaredModel}}"
+          pradeo.device.lastConnection: "{{json_event.message.content.deviceCorrelation.device.enrollmentStatus.lastConnection}}"
+          pradeo.device.coupled: "{{json_event.message.content.deviceCorrelation.device.enrollmentStatus.coupled}}"
+          pradeo.device.mdmId: "{{json_event.message.content.deviceCorrelation.device.emmDeviceInfo.externalId}}"
+          pradeo.device.emm: "{{json_event.message.content.deviceCorrelation.device.emmDeviceInfo.emm}}"
+          pradeo.policy.id: "{{json_event.message.content.deviceCorrelation.detectionPolicy.id}}"
+          pradeo.policy.name: "{{json_event.message.content.deviceCorrelation.detectionPolicy.name}}"
```

ECS fields [`rule.id`](https://www.elastic.co/guide/en/ecs/8.11/ecs-rule.html#field-rule-id) and [`rule.name`](https://www.elastic.co/guide/en/ecs/8.11/ecs-rule.html#field-rule-name) seems more suitable:
```suggestion
          rule.id: "{{json_event.message.content.deviceCorrelation.detectionPolicy.id}}"
          rule.name: "{{json_event.message.content.deviceCorrelation.detectionPolicy.name}}"
```